internal/server/settings.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/infrahq/infra/api"
+	"github.com/infrahq/infra/internal/access"
+	"github.com/infrahq/infra/internal/server/models"
+)
+
+func (a *API) GetSettings(c *gin.Context, r *api.GetSettingsRequest) (*api.Settings, error) {
+	res := api.Settings{}
+
+	if r.PasswordRequirements {
+		requirements, err := access.GetPasswordRequirements(c)
+		if err != nil {
+			return nil, err
+		}
+		res.PasswordRequirements = requirements
+	}
+
+	return &res, nil
```

Comment:
Should we just return all the setting when something specific isn't specified, rather than an empty response

Reply:
done